Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -117,11 +117,9 @@ Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	Considered concussive (explosive) damage. Hits armor points 
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
+	If armor roll is greater than 50, target rolls a DC 50 
 fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
-on stealth attacks. Reduces door-breaking DC by 30 per hit.
+on stealth attacks. Reduces break DCs of doors by 30 per hit.

```

Comment:
"break DCs" is super awk wording. Re my conventions this should become "the DC to break down a target door"